geode-core/src/main/java/org/apache/geode/distributed/internal/AbstractDistributionConfig.java

```diff
@@ -641,8 +640,8 @@ protected int checkSecurityLogLevel(int value) {
   @ConfigAttributeChecker(name = MEMCACHED_PROTOCOL)
   protected String checkMemcachedProtocol(String protocol) {
     if (protocol == null
-        || !protocol.equalsIgnoreCase(GemFireMemcachedServer.Protocol.ASCII.name())
-            && !protocol.equalsIgnoreCase(GemFireMemcachedServer.Protocol.BINARY.name())) {
+        || !protocol.equalsIgnoreCase("ASCII")
```

Comment:
Here we are changing from a type to a string value. In `DistributionConfig.java` below, we define a `DEFAULT_MEMCACHED_PROTOCOL` as that static string value. Should we be using a static String or Enum to hold this across all files?

Reply:
The enum is part of the geode-memcached subproject now, so we can't reference it in geode-core. It's also unfortunately public API and a subclass of GeodeMemcahedServer, so we can't reasonably leave it in core either.